src/yunohost/app.py

```diff
@@ -896,7 +909,14 @@ def confirm_install(confirm):
     os.system('chown -R root: %s' % app_setting_path)
     os.system('chown -R admin: %s/scripts' % app_setting_path)
 
-    app_ssowatconf(auth)
+    # Add path in permission if it's defined in the app install script
+    app_settings = _get_app_settings(app_instance_name)
+    domain = app_settings['domain']
```

Comment:
Yes like that ?
```suggestion
    if app_settings['domain']:
        domain = app_settings['domain']
```

Reply:
Was more thinking about `domain = app_settings.get("domain", None)` (otherwise `domain` might not be defined later and you are in trouble). Again, pretty small edge case anyway